VideoAPI/VideoApi.Domain/Conference.cs

```diff
@@ -180,9 +180,13 @@ public IList<InstantMessage> GetInstantMessageHistory()
         {
             return InstantMessageHistory.OrderByDescending(x => x.TimeStamp).ToList();
         }
-        public void AddInstantMessage(string from, string messageText)
+        public IList<InstantMessage> GetInstantMessageHistoryFor(string participantName)
```

Comment:
There doesn't seem to be a unit test in the domain folder for this method.

`VideoAPI/VideoApi.UnitTests/Domain/Conference`

Reply:
add a test for the domain method.

Reply:
Could we add a negative test also (i.e. non-existent username)

Reply:
added a test for non -existent participant